test/acceptance/authz/roles_test.go

```diff
@@ -723,3 +723,204 @@ func TestAuthzRoleRemoveToEmptyAndAddPermission(t *testing.T) {
 		require.Equal(t, authorization.CreateCollections, *role.Permissions[0].Action)
 	})
 }
+
+func TestAuthzRoleScopeMatching(t *testing.T) {
+	var err error
+
+	// Setup users
+	adminUser := "admin-user"
+	adminKey := "admin-key"
+	adminAuth := helper.CreateAuth(adminKey)
+
+	limitedUser := "limited-user"
+	limitedKey := "limited-key"
```

Comment:
could you use custom-user and custom-key to keep in it line with the docker-compose file? Then it is easier to switch between them for debugging :)